platform/darwin/test/MGLStyleTests.mm

```diff
@@ -99,14 +99,6 @@ - (void)testVersionedStyleURLs {
                           @(mbgl::util::default_styles::satelliteStreets.url));
     XCTAssertEqualObjects([MGLStyle satelliteStreetsStyleURLWithVersion:99].absoluteString,
                           @"mapbox://styles/mapbox/satellite-streets-v99");
-    XCTAssertEqualObjects([MGLStyle trafficDayStyleURLWithVersion:mbgl::util::default_styles::trafficDay.currentVersion].absoluteString,
-                          @(mbgl::util::default_styles::trafficDay.url));
```

Comment:
We should continue to test these versioned methods as long as they’re part of the API. To work around the deprecation warnings, [use pragmas](https://github.com/mapbox/mapbox-gl-native/blob/907a96a8f79491f39471f44218682c7f43b0ede9/platform/darwin/test/MGLStyleTests.mm#L68-L72).